ethereum-consensus/src/deneb/polynomial_commitments.rs

```diff
@@ -0,0 +1,152 @@
+use crate::{deneb::blob_sidecar::Blob, primitives::Bytes32, ssz::prelude::*};
+use c_kzg::Error;
+pub use c_kzg::KzgSettings;
+
+use super::mainnet::FIELD_ELEMENTS_PER_BLOB;
+
+pub const BYTES_PER_FIELD_ELEMENT: usize = 32;
+pub const BYTES_PER_BLOB: usize = BYTES_PER_FIELD_ELEMENT * FIELD_ELEMENTS_PER_BLOB;
+pub const BYTES_PER_COMMITMENT: usize = 48;
+pub const BYTES_PER_PROOF: usize = 48;
+
+pub type Bytes48 = ByteVector<48>;
```

Comment:
I don't see a reason to export this `Bytes48` type

and instead I'd prolly just do `pub type Kzg{Commitment, Proof} = ByteVector<48>;`

Reply:
use the constant as the length

Reply:
What exactly is this?   
`pub type Kzg{Commitment, Proof} = ByteVector<48>;`

Reply:
just a shorthand so I don't have to type as much


https://www.gnu.org/software/bash/manual/html_node/Brace-Expansion.html